argilla/tests/integration/test_export_dataset.py

```diff
@@ -175,11 +178,19 @@ def test_import_dataset_from_hub(
                 match="Trying to load a dataset `with_records=True` but dataset does not contain any records.",
```

Comment:
should we extend this test with passing a `workspace` too?

Reply:
We could in a subsequent PR.